Sources/App/configure.swift

```diff
@@ -51,9 +51,7 @@ public func configure(_ config: inout Config, _ env: inout Environment, _ servic
 
     let router = EngineRouter.default()
     try routes(router: router, slack: slack, commands: [
-        .fastlane(ci),
-        .hockeyapp(ci),
-        .testflight(ci),
+        .stevenson(ci),
         .crp(jira, github)
```

Comment:
Why is `.crp` not also included as `$ stevenson` subcommand?

Reply:
we can

Reply:
And we actually should since "crp" is such a generic term that it might one day also be a command name that other teams like BE or web might want to add to our slack for their own bot and own use case different than Stevenson 👍

We should again be sure to update all the related documentation accordingly though (playbook and wiki) too of course.